.travis.yml

```diff
@@ -1,40 +1,42 @@
 language: c
 env:
   global:
-    - GAPROOT=gaproot
-    - COVDIR=coverage
     - GAP_PKGS_TO_BUILD="json io profiling"
+    - TESTSCRIPT=scripts/run_tests.sh
 
 addons:
   apt_packages:
     - libgmp-dev
     - libreadline-dev
-    - libgmp-dev:i386
-    - libreadline-dev:i386
-    - gcc-multilib
-    - g++-multilib
+    - zlib1g-dev
     - python
     - python-pip
 
 matrix:
   include:
-    - env: CFLAGS="-O2" CC=clang CXX=clang++ 
-      compiler: clang
-    - env: CFLAGS="-O2"
-      compiler: gcc
-    - env: ABI=32
-    - env: CFLAGS="-O2" TESTCLIENTSERVER="yes"
-      compiler: gcc
+    - env: GAPBRANCH=stable-4.10
+    - env: GAPBRANCH=master
+    - env: GAPBRANCH=master ABI=32
+      addons:
+        apt_packages:
+          - libgmp-dev:i386
+          - libreadline-dev:i386
+          - zlib1g-dev:i386
+          - gcc-multilib
+          - g++-multilib
+          - python
+          - python-pip
```

Comment:
Personally, I'd just drop that 32bit test... up to you, though :-)

Reply:
I'm tempted to leave it for the time being – we'll see if I feel the same one day when it starts failing! ;)